### DISCUSSION TITLE
Profiling: Feedback and Ideas

### DISCUSSION THREAD

**Discussion by tonyo** [getsentry/sentry-go#657] [General]
Starting from [v0.22.0](https://github.com/getsentry/sentry-go/releases/tag/v0.22.0), the SDK now has experimental [profiling](https://docs.sentry.io/product/profiling/) support.

In this discussion people can leave feedback and suggestions for the feature.

![image](https://github.com/getsentry/sentry-go/assets/1120468/0d5b1558-9141-4605-b700-863d439cb6af)

**Comment by GlebBeloded** [comment]
Why did you remove profiling for go? :(
